src/pages/recurring-expenses/components/Details.tsx

```diff
@@ -88,7 +88,6 @@ export function Details(props: Props) {
             onClearButtonClick={() => handleChange('client_id', '')}
             onChange={(client) => handleChange('client_id', client.id)}
             errorMessage={errors?.errors.client_id}
-            staleTime={Infinity}
```

Comment:
@beganovich @turbo124 Setting staleTime to Infinity manually for the Create/Edit  Recurring Expense pages is no longer necessary because we have already changed it to be set by default.